### ISSUE TITLE
Micro 8088 default turbo option

### ISSUE THREAD

**Issue by TzOk83** [640-KB/GLaBIOS#25]
I've noticed that the default CPU clock set for Micro 8088 is 9.54 MHz, 4 RAM WS and this is not good, as it will fail on boards with 14.3MHz quartz. 7.15 MHz, 0 RAM WS would be much safer.

Is there any option to switch the "Turbo Mode" ON and OFF?

**Comment by 640-KB** [comment]
Yes, the speed is hot-key switchable during run time with the ctrl-alt-+ key stroke, which toggles between the default speed of 9.54MHz, 4 RAM WS and 7.15 MHz, 0 RAM WS.

This behavior is in fact customizable by changing `TURBO_TYPE = TURBO_REV` to `TURBO_TYPE = TURBO_STD` in the [Configuration for micro_8088](https://github.com/640-KB/GLaBIOS/blob/9f9c90ec5216588bed686207a339a272f4181251/src/GLABIOS.ASM#L319), which will make the startup speed be the "non-Turbo" 7.15 MHz, 0 RAM WS and the ctrl-alt-+ speed be the 9.54MHz, 4 RAM WS.  

Alternately the `FE_CPU_CFG` value can be [customized too](https://github.com/640-KB/GLaBIOS/blob/9f9c90ec5216588bed686207a339a272f4181251/src/GLABIOS.ASM#L305) for the default speed if it was ever desired to be something other than the current default of `011b` for 7.15 MHz, 0 RAM WS or `100b` for 9.54MHz, 4 RAM WS.

Your suggestion makes a lot of sense though to have an option so that boards with the slower quartz will work properly, which could look like either a separate image - perhaps the high-speed boot in the "high" image and the low-speed boot in the "low" image or possibly a configurable byte in the Flash image.  Would that do the trick?

Thanks for the testing and the suggestion!  Let me work on this a bit and I'll post an updated image.

**Comment by TzOk83** [comment]
Actually, for me, after changing crystal to 28.64MHz it switches between 9.54MHz and 4.77MHz, but that's completely fine. I may or may not have sth to do with the fact I'm using a Proton PT8010AF chipset, as the Faraday FE4010A is impossible to buy.

**Comment by 640-KB** [comment]
That's interesting. Perhaps the PT8010AF has slightly different speed register settings.  How does it work on the micro_8088 BIOS?

**Comment by TzOk83** [comment]
On Micro8088 BIOS I have all 3 speeds (4.77MHz, 7.15MHz & 9.54MHz) available. I have verified clock frequency, with a frequency counter.

**Comment by 640-KB** [comment]
I just looked back at the code and toggling between highest and lowest (4.77) was what I intended (I just forgot that's how it worked).  I'll look into what it would take to add three state toggling for it.
